packages/react-slider/src/components/Slider/stories/SliderDisabled.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Label } from '@fluentui/react-label';
+import { useId } from '@fluentui/react-utilities';
+import { Slider } from '../../../index';
+
+export const Disabled = () => {
+  const id = useId();
+  return (
+    <>
+      <Label htmlFor={id}>Disabled Example</Label>
+      <Slider defaultValue={30} disabled id={id} />
+    </>
+  );
+};
```

Comment:
These stories should have descriptions to be like the other stories in storybook.

Reply:
fixed